src/main/kotlin/no/nav/syfo/domain/ManglendeMedvirkningVurdering.kt

```diff
@@ -137,17 +158,17 @@ data class Varsel(
 
 fun VurderingType.dokumentTittel(): String = when (this) {
     VurderingType.FORHANDSVARSEL -> "Forhåndsvarsel om stans av sykepenger"
-    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> "Vurdering av § 8-8 manglende medvirkning"
+    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL, VurderingType.UNNTAK -> "Vurdering av § 8-8 manglende medvirkning"
     VurderingType.STANS -> "Innstilling om stans"
 }
 
 fun VurderingType.brevkode(): BrevkodeType = when (this) {
     VurderingType.FORHANDSVARSEL -> BrevkodeType.MANGLENDE_MEDVIRKNING_FORHANDSVARSEL
-    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> BrevkodeType.MANGLENDE_MEDVIRKNING_VURDERING
+    VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL, VurderingType.UNNTAK -> BrevkodeType.MANGLENDE_MEDVIRKNING_VURDERING
     VurderingType.STANS -> BrevkodeType.MANGLENDE_MEDVIRKNING_STANS
 }
 
 fun VurderingType.journalpostType(): JournalpostType = when (this) {
-    VurderingType.FORHANDSVARSEL, VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL -> JournalpostType.UTGAAENDE
+    VurderingType.FORHANDSVARSEL, VurderingType.OPPFYLT, VurderingType.IKKE_AKTUELL, VurderingType.UNNTAK -> JournalpostType.UTGAAENDE
```

Comment:
Avklart med Stine og co at alle disse skal sendes ut? 😊

Reply:
Ja, vi mener det har blitt riktig. Men STANS bør ha NAY som mottaker (som for § 8-4)